py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -1,14 +1,27 @@
-from typing import Optional, cast
+import logging
+from typing import Optional, Sequence, Union, cast
 
 import numpy as np
+import tensorrt as trt
 from torch.fx.node import Target
 from torch_tensorrt.dynamo._SourceIR import SourceIR
 from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
-from torch_tensorrt.dynamo.conversion.converter_utils import get_positive_dim, to_numpy
+from torch_tensorrt.dynamo.conversion.converter_utils import (
+    broadcastable,
+    get_trt_tensor,
+)
+from torch_tensorrt.dynamo.conversion.impl.elementwise import convert_binary_elementwise
 from torch_tensorrt.dynamo.conversion.impl.shape import get_shape_with_dynamic_shape
-from torch_tensorrt.fx.converters.converter_utils import has_dynamic_shape
+from torch_tensorrt.fx.converters.converter_utils import (
+    get_positive_dim,
+    has_dynamic_shape,
+    set_layer_name,
+    to_numpy,
+)
```

Comment:
Switch `get_positive_dim` and `to_numpy` to the `torch_tensorrt.dynamo.conversion.converter_utils` version